.pre-commit-config.yaml

```diff
@@ -271,7 +271,11 @@ repos:
     - tests/
 
 - repo: https://github.com/pycqa/pylint.git
-  rev: v2.12.2
```

Comment:
And the big challenge would be on how to prevent pre-commit to bump this. We will no longer be able to run `pre-commit autoupdate` without undoing this change.

I am wondering if that issue is not related to an outdated pip because if we ping astroid and pylint asks for something else pip should fail to install. AFAIK, that works fine with recent pip versions.

Reply:
I am inclined to believe that silencing no-member error until we get a fix would be easier.